code/modules/lighting/lighting_object.dm

```diff
@@ -103,19 +103,18 @@
 	//anything that passes the first case is very likely to pass the second, and addition is a little faster in this case
 		icon_state = "transparent"
 		color = null
+	else if(!set_luminosity)
+		icon_state = "dark"
+		color = null
```

Comment:
I just checked and actually it does, just setting the color to null and letting the luminosity get set to 0 actually renders tiles as those glitched light overlays from https://github.com/tgstation/tgstation/pull/25654#issuecomment-290594869 . The matrix has to use some trickery to fulldark that tile, which is why something that should have been simple was instead costly.